### PR TITLE
add logging

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

- add general logging
- add loggin when loading the model

Fixes #

## How Has This Been Tested?

CI

- [ ] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: pv_site_production/models/psp.py [hunk 1]
```diff
@@ -11,17 +11,20 @@
 
 from pv_site_production.utils.imports import instantiate
 
+_log = logging.getLogger()
```
Comment (simlmx) [inline pv_site_production/models/psp.py:14]:
```suggestion
_log = logging.getLogger(__name__)
```

File: pv_site_production/app.py [hunk 2]
```diff
@@ -18,6 +18,10 @@
 from pv_site_production.utils.config import load_config
 from pv_site_production.utils.imports import import_from_module
 
+logging.basicConfig(
+    level=getattr(logging, os.getenv("LOGLEVEL", "DEBUG")),
```
Comment (simlmx) [inline pv_site_production/app.py:22]: We could consider using a click argument for the log level, but this is fine too.

File: pv_site_production/models/psp.py [hunk 3]
```diff
@@ -11,17 +11,20 @@
...
 
+    _log.debug(f'Getting NWP data: {config["nwp"]}')
     nwp_data_source = instantiate(**config["nwp"])
 
     # TODO Make the setup step uniform across all `psp` models. In other words it should be defined
     # directly in `PvSiteModel`.
-    model.setup(
-        SetupConfig(pv_data_source=pv_data_source, nwp_data_source=nwp_data_source)
-    )
+    model.setup(SetupConfig(pv_data_source=pv_data_source, nwp_data_source=nwp_data_source))
```
Comment (simlmx) [inline pv_site_production/models/psp.py:28]: I suspect your editor is not using the settings from the repo or something, if the formatting is changing here!

File: pv_site_production/app.py (hunk 2 above)
Comment (peterdudfield) [reply pv_site_production/app.py:22]: yea, ill leave this for the moment

File: pv_site_production/models/psp.py (hunk 3 above)
Comment (peterdudfield) [reply pv_site_production/models/psp.py:28]: i just ran blacks set to line length 100. Maybe its not set to that

Comment (peterdudfield) [reply pv_site_production/models/psp.py:28]: ill move it back

Comment (simlmx) [reply pv_site_production/models/psp.py:28]: Only flake8 is set to 100: https://github.com/openclimatefix/pv-site-prediction/blob/main/.flake8#L2
Black probably still has the default setting then, we should probably fix that. Doesn't matter for now I guess.

**Pull request merged**